### PR TITLE
Organize styles

### PR DESCRIPTION
**Pull request by brycedorn**
This pull request includes a

- [ ] Bug fix
- [x] New feature
- [ ] Translation

The following changes were made

- `static/css/style8.css` => `static/sass/components/_overlay.scss`
- Got rid of inline styles in `templates/map.html`
- Moved import for select2 lib to `main.scss`

~~If this is related to an existing ticket, include a link to it as well.~~

### CODE REVIEW

**Comment by AHAAAAAAA** [comment]
+1s all around, good job.

**Comment by CarlTaylor1989** [comment]
+1

**Comment by medghaim** [comment]
+1

**Comment by medghaim** [comment]
pffft, fine

File: static/css/main.css [hunk 1]
```diff
@@ -1921,7 +2014,8 @@ body {
 #map {
   top: 3.5em;
   bottom: 0px;
-  width: 100%; }
+  height: 100vh;
```
Comment (MRokas) [inline static/css/main.css]: Change this back to 100%, because it ruins zoom control now.

Comment (brycedorn) [inline static/css/main.css]: thanks for taking care of it - looks like this carried over from the original `#map` styles after merging

Comment (HenrySia) [inline static/css/main.css]: @brycedorn I can't see the zoom-in button on google chrome desktop version.

Comment (brycedorn) [inline static/css/main.css]: @HenrySia pull `develop` again, was fixed with https://github.com/AHAAAAAAA/PokemonGo-Map/pull/1221

Comment (HenrySia) [inline static/css/main.css]: @brycedorn No #1221 didn't fix it. #1277 may did it, but I haven't tried.

On a side note, thanks for the good work you've done!

Comment (MRokas) [inline static/css/main.css]: #1221 did fix it. If you are unable to properly build files, dont blame me.

Comment (HenrySia) [inline static/css/main.css]: @MRokas I'm not blaming you. Either someone broke this again, or your fix didn't work, I don't know. It's just what it is.

I did a clean git clone -b develop (468b64252988afc8932574c492fef5e7c55997db), and start the runserver.py. Open my web browser and the zoom out button got cropped, thus can't be clicked.

Comment (MRokas) [inline static/css/main.css]: It worked for me and others. Did you build files or not?

Comment (HenrySia) [inline static/css/main.css]: @MRokas 20fcf1fb4573bb93cf896e3a38ecab25a2af73f1 fixed it. Thanks!

**Pull request merged**